src/dotnet/Fable.Tools/Server.fs

```diff
@@ -41,6 +41,7 @@ let rec private loop timeout (server: TcpListener) (buffer: byte[]) (onMessage:
             if data = SIGTERM
             then
                 Log.logAlways("Closing Fable daemon...")
+                Environment.Exit 1
```

Comment:
maybe instead of that,

```
Environment.Exit 1
return ()
```

to force exit, just do

```
return 1
```

i seen the flow, and error code is returned for all (from `p.ExitCode`), minus this part